spec/features/18f_procurement_spec.rb

```diff
@@ -27,7 +27,7 @@
       expect(Dispatcher).to receive(:deliver_new_proposal_emails)
 
       visit '/gsa18f/procurements/new'
-      fill_in "Product Name and Description", with: "buying stuff"
+      fill_in "Product name and description", with: "buying stuff"
```

Comment:
Not sure I understand what you're asking? I was just noticing that we had to touch a lot of code (in specs) to make a small copy change, didn't intend to comment on the change itself